js/ssai_pulse.js

```diff
@@ -180,6 +190,7 @@ OO.Ads.manager(function(_, $)
               isLive: this.currentAd.ad.isLive
             }
           );
+          _handleTrackingUrls(this.currentAd, ["impression", "start"]);
```

Comment:
We should handle the impression URL before notify linear ad started. This causes some messages to get fired which may cause some state issues down the road.